woocommerce/woocommerce-android
# Domain change: Current domain screen

Part of #8238.

<img src="https://user-images.githubusercontent.com/1522856/214870101-77c1da5c-927d-4981-95aa-a11793b9bf01.png" width="300" />

This PR adds the current domain(s) screen barebone UI, which allows the users to:

- Display the free WP.com domain
- Claim a free domain
- Display a list of existing domains associated with the store
- Search for a new domain

**To test:**
1. Navigate to the preferences screen
2. Tap on the Domains item
3. Verify the current domain screen is displayed, showing the current site address as a free domain and some dummy paid domains
